### PR TITLE
[4697] change genQuery SELECT to select (4-2-stable)

### PR DESCRIPTION
**Pull request by trel**
this is a workaround until the new parser lands

test passes

### CODE REVIEW

**Review by alanking — APPROVED** [review]
Other tests also pass?

File: scripts/irods/test/test_all_rules.py [hunk 1]
```diff
@@ -1157,3 +1157,8 @@ def test_rename_to_existing_collection_with_different_name_is_an_error__issue_54
         cmd = ['irule', '-r', rep_name, 'msiRenameLocalZoneCollection("{0}")'.format(existing_collection), 'null', 'ruleExecOut']
         self.admin.assert_icommand(cmd, 'STDERR', ['-809000 CATALOG_ALREADY_HAS_ITEM_BY_THAT_NAME'])
 
+    @unittest.skipUnless(plugin_name == 'irods_rule_engine_plugin-irods_rule_language', 'only applicable for irods_rule_language REP')
+    def test_use_lowercase_select_in_genquery_conditions__issue_4697(self):
+        rep_name = 'irods_rule_engine_plugin-irods_rule_language-instance'
+        cmd = ['irule', '-r', rep_name, '-F', '/var/lib/irods/clients/icommands/test/rules/rulemsiMakeGenQuery.r', "*Coll='/tempZone/home/rods/select/%%'"]
```
Comment (korydraughn) [inline scripts/irods/test/test_all_rules.py:1163]: why two percent trailing signs on the `*Coll` argument?

**Review by korydraughn — APPROVED** [review]

File: scripts/irods/test/test_all_rules.py (hunk 1 above)
Comment (trel) [reply scripts/irods/test/test_all_rules.py]: hrm.  that's what is in the rulefile itself...   and was the example that triggered this issue.

I've dropped it to a single `%` and am now using a testfile to match as well.

**Comment by trel** [comment]
CI running.

**Comment by trel** [comment]
all tests passed.

**Pull request merged**